packages/block-library/src/list/edit.js

```diff
@@ -41,6 +41,7 @@ export default function ListEdit( {
 	setAttributes,
 	mergeBlocks,
 	onReplace,
+	mergedStyle,
```

Comment:
Why not simply "style"?

Reply:
Thanks for the suggestion @ellatrix and it does make more sense to be named `style`, I renamed these to be more consistent. https://github.com/WordPress/gutenberg/pull/30997/commits/ed6e9d2021b2ffa84bb34988ca7a816e3eae1107